pkg/server/configuration.go

```diff
@@ -0,0 +1,76 @@
+package server
+
+import (
+	"net/http"
+	"path/filepath"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pluralsh/plural/pkg/manifest"
+	"github.com/pluralsh/plural/pkg/utils/git"
+)
+
+type Configuration struct {
```

Comment:
One thing i just remembered, we should also return the current context configuration here as well, probably just as a `map[string]interface{}`

Reply:
OK, no problem

Reply:
done